core/services/chainlink/testdata/config-full.toml

```diff
@@ -0,0 +1,274 @@
+Dev = true
```

Comment:
You may have mentioned this elsewhere but perhaps this belongs under a `[Global]` or `[Chainlink]` or `[Core]` section?

Reply:
I thought it might be necessary in order to keep the type imports sane, but that turns out not to be the case (can just embed). Can still consider this for UX though if it is more clear for everything to be under a grouping :shrug: